### PR TITLE
Feat/geometric computations

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: bindings/python/src/geometry/geometry.cpp [hunk 1]
```diff
@@ -24,9 +24,15 @@
 #include <pybind11/pybind11.h>
```
Comment (panquez) [inline bindings/python/src/geometry/geometry.cpp]: Missing header

Comment (BotellaA) [reply bindings/python/src/geometry/geometry.cpp]: No. There is a header! ;)

Comment (panquez) [reply bindings/python/src/geometry/geometry.cpp]: Oh my bad

**Pull request merged**